evm/chains/sharding/mainchain_handler/vmc_utils.py

```diff
@@ -0,0 +1,135 @@
+import functools
+import json
+import os
+
+from cytoolz import (
+    pipe,
+)
+
+import rlp
+
+from eth_keys import (
+    keys,
+)
+
+from eth_utils import (
+    to_canonical_address,
+    to_tuple,
+    keccak,
+)
+
+from evm.utils.address import (
+    generate_contract_address,
+)
+from evm.utils.hexadecimal import (  # use evm.utils over eth_utils for hex encode/decode
+    decode_hex,
+)
+
+from evm.chains.sharding.mainchain_handler.config import GASPRICE
+
+WITHDRAW_HASH = keccak("withdraw")
```

Comment:
This should be changed to `keccak(b"withdraw")` as well as changing to use `evm.utils.keccak.keccak` which only accepts bytes inputs.  Reason being that `evm.utils.keccak` is going to change in the near term future in a breaking way that will cause this way of calling it to break.

Reply:
No problem I will change it. Thank you:)